Modules/Package.swift

```diff
@@ -71,7 +79,7 @@ let package = Package(
         .package(url: "https://github.com/squarefrog/UIDeviceIdentifier", from: "2.3.0"),
         .package(url: "https://github.com/stripe/stripe-terminal-ios", from: "4.2.0"),
         .package(url: "https://github.com/SVProgressHUD/SVProgressHUD", from: "2.2.5"),
-        .package(url: "https://github.com/wordpress-mobile/AztecEditor-iOS", from: "1.20.0"),
+        .package(url: "https://github.com/wordpress-mobile/AztecEditor-iOS", revision: "d741e3cfaa74c99ef092e5fddb87d4314b63e3ed"),
```

Comment:
This was necessary to compile NetworkingCore without UIKit, see https://github.com/wordpress-mobile/AztecEditor-iOS/pull/1412#

Reply:
> Can we do this with an Aztec release?
Yes.

> I thought that was part of the development process now, rather than app release...?
Correct. We don't need to be too strict about it, as in "you _must_ ship a new version before using library changes in the app" but it'd be good practice to have something stable and tagged.

The changes in that commit came from https://github.com/wordpress-mobile/AztecEditor-iOS/pull/1412 which was motivated by the modularization work here. At the time, the PR hadn't been merged, but since it has now, I shall follow up with a release. 

Thanks!